nearlib/near.js

```diff
@@ -1,8 +1,46 @@
+const BSON = require('bsonfy').BSON;
+const NearBase = require('./nearbase');
+
 /*
- * This is the javascript library for using near blockchain.
- * This is a placeholder for now.
+ * This is javascript library for interacting with blockchain.
  */
+class Near extends NearBase {
```

Comment:
Why is this class needed vs having these methods in `Account`? Methods of this class operate on specific account level (they require sender).

Reply:
It's not 100% clear at this point what libraries should be there and what are the lines of separation. This said, it's useful to have 2 js files immediately to be able to place common functionality in an appropriate place rather just in _the single js file_. I am 100% sure stuff will move around in the future and be refactored, but I suggest to leave this temporary separation be for now as a forcing function for some other code structure matters.

Reply:
(good news is that refactoring is trivial with fast tests :) )

Reply:
The file that is split here was less than 100 lines long. Not sure why split at this point.

Also still nor clear why have methods operating on specific account outside of `Account`. I don't expect this as an API user.

Reply:
My concern isn't about common code structure. Common code is e.g. `NearClient`, I'm ok with that, doesn't matter much where it goes.

My concern is about API surface we expose. Right now we have some methods operating on account in `Account` class and some in `Near` class. I cannot see why API user should use two different classes here. 

I see `Account` object as API for everything account-scoped. There can be other objects for more narrow scopes, e.g. for a given `Account` you can get some `Contract` object which will only give access to specific contract methods.

Reply:
Do stuff with my account vs do stuff with a contract separation is also something that makes sense. In any case, as I said before --- it's premature to make this particular design decision. I want to stick to 2 files to help structure the code well, and once we have a better feel of what's a good API separation, it will be trivial to rename things/ move functions around.